config/kdump.py

```diff
@@ -1,44 +1,69 @@
-import os
+import sys
+
 import click
 import utilities_common.cli as clicommon
-from swsscommon.swsscommon import ConfigDBConnector
+from swsssdk import ConfigDBConnector
```

Comment:
>swsssdk [](http://example.com/codeflow?start=5&length=7)

swsssdk is deprecated. Please use libswsscommon instead.

Reply:
Updated!